src/arrayMethodSort.js

```diff
@@ -5,7 +5,41 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    const compareAsStrings = (a, b) => {
+      const stringA = String(a);
+      const stringB = String(b);
+
+      if (stringA > stringB) {
+        return 1;
+      }
+
+      if (stringA < stringB) {
+        return -1;
+      }
+
+      return 0;
+    };
```

Comment:
will be better to move this function outside of the sort2 method and set it by default while describing function parameters